### PR TITLE
Gdpr compliance

### LINKED ISSUE #105: Add GDPR Compliance
We should have a banner telling our users that we're using cookies.

### LINKED ISSUE #107: Add Privacy Policy
Eric needs to sculpt a Privacy Policy for our site to meet GDPR requirements

For reference: https://www.trailheadai.com/privacy-policy

### PR DESCRIPTION
**Pull request by sw00d**
resolves #107 and resolves #105 

To test:
- [x] Delete all cookies and visit site
- [x] A cookie-consent banner should show up at the bottom that you can close
- [x] Refresh page and banner should not re-appear
- [x] Scroll to bottom and click Privacy Policy in Footer
- [x] Scroll through Privacy Policy page and look for anything obviously wrong.

### CODE REVIEW

**Comment by ckcollab** [comment]
Can you make clicking anywhere on that cookie thing close it? may be a bad call, will let you decide if that's dumb or not :D

File: about_us.html [hunk 1]
```diff
@@ -185,14 +184,18 @@
...
+                <a href="/privacy-policy.html" class="nav-item">Privacy Policy</a>
+            </div>
             <div class="blue-stripe-lower"></div>
         </section>
     </v-app>
 </div>
 </body>
 </html>
 
+<script src="./assets/js/cookie-consent.js" type="text/javascript"></script>
```
Comment (ckcollab) [inline about_us.html:202]: do we need this import repeated so many times? may be so..

Comment (sw00d) [reply about_us.html:202]: Yeah, unless I'm missing something but we need to have that banner on every page. If there's a better way to do this let me know.

Comment (ckcollab) [reply about_us.html:202]: Yeah that's what I was afraid of. Would be neat if we had a hook/template of some kind so we didn't have to do this.. oh well, works for now!

**Pull request merged**